sklearn/neighbors/_classification.py

```diff
@@ -107,25 +112,21 @@ class KNeighborsClassifier(NeighborsBase, KNeighborsMixin,
         False when `y`'s shape is (n_samples, ) or (n_samples, 1) during fit
         otherwise True.
 
-    Examples
-    --------
-    >>> X = [[0], [1], [2], [3]]
-    >>> y = [0, 0, 1, 1]
-    >>> from sklearn.neighbors import KNeighborsClassifier
-    >>> neigh = KNeighborsClassifier(n_neighbors=3)
-    >>> neigh.fit(X, y)
-    KNeighborsClassifier(...)
-    >>> print(neigh.predict([[1.1]]))
-    [0]
-    >>> print(neigh.predict_proba([[0.9]]))
-    [[0.66666667 0.33333333]]
+    Methods
```

Comment:
We don't list the methods normally. It will be done automatically by Sphinx when compiling the documentation.